db/migrate/20200804170036_add_alias_to_institutions.rb

```diff
@@ -0,0 +1,5 @@
+class AddAliasToInstitutions < ActiveRecord::Migration[5.2]
```

Comment:
This is not needed

Reply:
I can pull those out, wasn't sure if we wanted it to be stored under `alias` as it comes in from scorecard or `ialias` as it currently is

Reply:
let's continue to use the existing column as `ialias` probably is just "institution alias"

also this reduces the code changes and database changes